docs/cunumeric/source/versions.rst

```diff
@@ -7,4 +7,5 @@ Versions
 
   22.01 <https://nv-legate.github.io/cunumeric/22.01>
   22.03 <https://nv-legate.github.io/cunumeric/22.03>
+  22.05 <https://nv-legate.github.io/cunumeric/22.05>
```

Comment:
@magnatelee : I hope you don't mind this change to come with this PR.

Reply:
Ok.